submissions/kolajoTomike/Javascript/Task2/app.js

```diff
@@ -0,0 +1,23 @@
+//Task 2//
+
+ const diff=  123-7;
+ var count=1;
+
+ while (count < diff){
```

Comment:
This statement should be less than and equal to @tomike19

Reply:
changes made